docs/zh/start/configuration/start.md

```diff
@@ -4,8 +4,23 @@
 本示例架构如下图，启动的进程有：客户端程程序、Layotto、etcd 。
 
 ![](https://gw.alipayobjects.com/mdn/rms_5891a1/afts/img/A*dzGaSb78UCoAAAAAAAAAAAAAARQnAQ)
+### step 1. 启动 etcd 和 Layotto
+<!-- tabs:start -->
+#### **使用 Docker Compose**
```

Comment:
Could u also modify the english version of this doc?
https://mosn.io/layotto/#/en/start/configuration/start

Reply:
no problem